packages/stencil/components/select/src/components/osds-select/osds-select.e2e.ts

```diff
@@ -260,4 +261,64 @@ console.log('before clicks');
       expect(await isSelectOpen()).toBe(false);
     });
   });
+
+  describe('method:handlerKeyPress', () => {
+    it('should open select when Enter', async () => {
+      await setup({ });
+      await page.waitForChanges();
+      await el.focus();
+      await page.keyboard.press('Enter');
+      expect(await el.getProperty('opened')).toBeTruthy();
```

Comment:
we should try to avoid `toBeTruthy` and `toBeFalsy` when we know the exact expected value cause it can pass with the wrong value
prefer `toBe('true')`